susemanager-utils/testing/docker/README.md

```diff
@@ -111,12 +102,12 @@ which runs on `registry.mgr.suse.de`.
 
 First you need to tag the local images:
 ```
-docker tag suma-<branch>-<base|pgsql|ora> registry.mgr.suse.de/suma-<branch>-<base|pgsql|ora>
-docker tag suma-<branch>-<base|pgsql|ora>:<version> registry.mgr.suse.de/suma-<branch>-<base|pgsql|ora>:<version>
+docker tag uyuni-<branch>-<base|pgsql> registry.mgr.suse.de/uyuni-<branch>-<base|pgsql>
+docker tag uyuni-<branch>-<base|pgsql>:<version> registry.mgr.suse.de/uyuni-<branch>-<base|pgsql>:<version>
 ```
 
 Then you can push the local image:
 ```
-docker push registry.mgr.suse.de/suma-<branch>-<base|pgsql|ora>
-docker push registry.mgr.suse.de/suma-<branch>-<base|pgsql|ora>:<version>
+docker push registry.mgr.suse.de/uyuni-<branch>-<base|pgsql>
```

Comment:
Maybe not for this PR, but we need to start thinking about a public registry (registry.opensuse.org?)

Reply:
yes, but this is for later. I think we should make it run with registry.mgr.suse.de first and then we check where we should push this to.